dev-libs/qtcompress/qtcompress-0_p20180513.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils toolchain-funcs virtualx
+
+DESCRIPTION="A public API for QZip in an easy to use module"
+
+HOMEPAGE="https://github.com/nezticle/qtcompress"
+MY_COMMIT="23f8831826cd72aedf99fc3699148b6c994fd677"
+SRC_URI="https://github.com/nezticle/qtcompress/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+S="${WORKDIR}/qtcompress-${MY_COMMIT}"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	sys-libs/zlib:=
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	test? ( dev-qt/qttest:5 )
+"
+
+PATCHES=(
+	"${FILESDIR}/${P}-remove-zlib.patch"
+	"${FILESDIR}/${P}-test-include.patch"
+)
+
+src_prepare() {
+	default
+	rm -r src/3rdparty/ || die
```

Comment:
Could you add a comment above the `rm` explaining what it does? To me it looks like it removes bundled lib code, but for anyone reading and wondering. People shouldn't be guessing, and then one day upstream might drop the dir and this package is m-n and someone looking at it thinking what was this for.